libdnf/rpm/solv/package-private.hpp

```diff
@@ -0,0 +1,451 @@
+/*
+Copyright (C) 2020 Red Hat, Inc.
+
+This file is part of libdnf: https://github.com/rpm-software-management/libdnf/
+
+Libdnf is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+Libdnf is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#ifndef LIBDNF_RPM_PACKAGE_PRIVATE_HPP
+#define LIBDNF_RPM_PACKAGE_PRIVATE_HPP
+
+
+#include "libdnf/rpm/package.hpp"
+#include "id_queue.hpp"
+
+extern "C" {
+#include <solv/pool.h>
+#include <solv/pooltypes.h>
+#include <solv/repo.h>
+#include <solv/solvable.h>
+#include <solv/util.h>
+}
+
+#include <cassert>
+#include <cstdlib>
+#include <string>
+#include <vector>
+
+namespace libdnf::rpm::solv {
+
+const char * BASE_EPOCH = "0";
```

Comment:
It is variable that points to const C string.
Probably you want to declare constant `constexpr const char * BASE_EPOCH = "0";`

Reply:
Added `constexpr`